tests/components/wmspro/test_init.py

```diff
@@ -0,0 +1,41 @@
+"""Test the wmspro initialization."""
+
+from unittest.mock import AsyncMock
+
+import aiohttp
+
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+async def test_config_entry_device_config(
+    hass: HomeAssistant,
+    mock_config_entry: MockConfigEntry,
+    mock_hub_ping: AsyncMock,
+    mock_hub_refresh: AsyncMock,
+) -> None:
+    """Test that a config entry can be loaded with DeviceConfig."""
+    mock_config_entry.add_to_hass(hass)
+    await hass.config_entries.async_setup(mock_config_entry.entry_id)
+    await hass.async_block_till_done()
+    assert mock_config_entry.state is ConfigEntryState.LOADED
```

Comment:
```suggestion
    mock_config_entry.add_to_hass(hass)
    assert await hass.config_entries.async_setup(mock_config_entry.entry_id)
    await hass.async_block_till_done()
```
This would test the same thing but a bit shorter.

Consider moving this to a helper function because you use this at every test